code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -184,4 +184,88 @@
 /datum/gear/head/zhan_scarf
 	display_name = "Zhan headscarf"
 	path = /obj/item/clothing/head/tajaran/scarf
-	whitelisted = "Tajara"
\ No newline at end of file
+	whitelisted = "Tajara"
+
+/datum/gear/head/hijab
+	display_name = "Black hijab"
+	path = /obj/item/clothing/head/hijab
+
+/datum/gear/head/hijab/white
+	display_name = "White hijab"
+	path = /obj/item/clothing/head/hijab/white
+
+/datum/gear/head/hijab/aqua
+	display_name = "Aqua hijab"
+	path = /obj/item/clothing/head/hijab/aqua
+
+/datum/gear/head/hijab/blue
+	display_name = "Blue hijab"
+	path = /obj/item/clothing/head/hijab/blue
+
+/datum/gear/head/hijab/brown
+	display_name = "Brown hijab"
+	path = /obj/item/clothing/head/hijab/brown
+
+/datum/gear/head/hijab/darkblue
+	display_name = "Dark blue hijab"
+	path = /obj/item/clothing/head/hijab/darkblue
+
+/datum/gear/head/hijab/darkred
+	display_name = "Dark red hijab"
+	path = /obj/item/clothing/head/hijab/darkred
+
+/datum/gear/head/hijab/green
+	display_name = "Green hijab"
+	path = /obj/item/clothing/head/hijab/green
+
+/datum/gear/head/hijab/green
+	display_name = "Green hijab"
+	path = /obj/item/clothing/head/hijab/grey
```

Comment:
I believe the copy-paste demon been involved, as I have a feeling this grey hijab isn't intended to be under the green entry.